megamek/data/mechfiles/mechs/Iron Wind Metals/Pendragon PDG-3R.mtf

```diff
@@ -1,154 +1,154 @@
-Version:1.0
-Pendragon
-PDG-3R
-
-Config:Biped
-TechBase:Inner Sphere
-Era:3130
-Rules Level:2
-Source:Icewind Metals

```

Comment:
The unit files also give the source as Icewind Metals.

Reply:
Whoops, fixing that now

Reply:
Fixed, and fixed my search parameters to also include file contents (thought I had it enabled on this computer, turns out I did not)